### ISSUE TITLE
some queries

### ISSUE THREAD

**Issue by pangda72** [CN-zdy/MSU_Net#15]
Hi,CN-zdy
I have some queries regarding your work? Colud you please give me a hint,thank you very much!

1、Why does testsets need to be tested more than once.In other words,why need num_epochs_test?

2、when i run 'train.py'(model type is UNet),my epoch is 300,but my average loss only dropped to about 0.15,I would like to know how low can the average loss go down in your experiment.Can the average loss be reduced to 0.00xx?

**Comment by CN-zdy** [comment]
Hi, Pangda.
Question 1: I would like to get a more accurate prediction result.
Question 2: The loss in my experiment is higher than yours.

**Comment by pangda72** [comment]
> Hi, Pangda.
> Question 1: I would like to get a more accurate prediction result.
> Question 2: The loss in my experiment is higher than yours.

Hi ,CN-zdy
Thank you for your answer.I get it.thanks a lot.

**Comment by pangda72** [comment]
Hi ,CN-zdy
Did you do any data augmentation  in your experiment?  Because I found hyperparameters(augmentation_prob) in the code,
But I couldn't find the augmentation part.Colud you please give me a hint,thank you very much!

**Comment by CN-zdy** [comment]
Hi, Pangda.
I used data augmentation in FSNet.
I didn't upload the code.
Because I'm lazy.

**Comment by pangda72** [comment]
> Hi, Pangda.
> I used data augmentation in FSNet.
> I didn't upload the code.
> Because I'm lazy.

Hi, CN-zdy
I was very sad to hear this news.You did a good job.As a beginner, your code is very helpful to me.I think you should keep updating FSNet as it is a good work and will help us a lot. Can you continue to update this part of the code?It means a lot to me. If you don't want to do this,could you  send it to my email:sun2818113235@163.com

Thanks you very much!
Best.

**Comment by CN-zdy** [comment]
Hi, Pangda.
The code has been sent to your email address.

**Comment by pangda72** [comment]
> Hi, Pangda.
> The code has been sent to your email address.

Hi CN-zdy
I have received your code, thank you for your guidance and advice, it will be a great help to me,I will try as you suggest.
Thank you once again.
Best wishes.
